doc/nrf/protocols/matter/end_product/index.rst

```diff
@@ -36,3 +36,4 @@ Finally, we discuss topics related to Matter certification (:ref:`ug_matter_devi
    configuring_cd
    versioning
    last_fabric_removal_delegate
+   test_event_triggers
```

Comment:
Should probably add a mention to Line 22 as well. Not sure if it should be with the bootloader mention, or inside the parenthesis with topics related to certification.